DICOM/.editorconfig

```diff
@@ -0,0 +1,8 @@
+﻿[*.cs]
+
+# CS3021: Type or member does not need a CLSCompliant attribute because the assembly does not have a CLSCompliant attribute
+dotnet_diagnostic.CS3021.severity = none
```

Comment:
Remove line 4 and line 8 here. We can provide a more useful editorconfig like [this](https://github.com/microsoft/fhir-server/blob/main/.editorconfig) or just remove this file